geomstats/geometry/matrices.py

```diff
@@ -128,7 +133,20 @@ def make_symmetric(cls, mat):
         return 1 / 2 * (mat + cls.transpose(mat))
 
     def random_uniform(self, n_samples=1):
-        """Generate n samples from a uniform distribution."""
+        """Generate n samples from a uniform distribution.
+
+        Each entry in the matrix is drawn from a uniform distribution with
+        support [0,1]
+
+        Parameters
+        ----------
+        n_samples : int
+            amount of samples to generate
```

Comment:
Should the first letter be upper-case or lower-case? I had used upper case, but happy to change my PR!